frontend-cert/js-projects/music-instrument-filter/script.js

```diff
@@ -0,0 +1,52 @@
+// have campers add the first 3 objects, then add the rest
+const instrumentsArr = [
+  { category: "woodwinds", instrument: "Flute", price: 500 },
+  { category: "woodwinds", instrument: "Clarinet", price: 200 },
+  { category: "woodwinds", instrument: "Oboe", price: 4000 },
+  { category: "brass", instrument: "Trumpet", price: 200 },
+  { category: "brass", instrument: "Trombone", price: 300 },
+  { category: "brass", instrument: "French Horn", price: 4300 },
+  { category: "percussion", instrument: "Drum Set", price: 500 },
+  { category: "percussion", instrument: "Xylophone", price: 3000 },
+  { category: "percussion", instrument: "Cymbals", price: 200 },
+  { category: "percussion", instrument: "Marimba", price: 3000 },
+];
+
+const selectContainer = document.querySelector("select");
+const productsContainer = document.querySelector(".products-container");
+
+// build out function in pieces
+// first start with return filtered arr. have them test it with console statements
```

Comment:
Once they have the initial log set up, we should just tell them to leave it so they can watch their changes unfold in "real time".

Reply:
Yeah, I agree.
That's a good point